artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/coreWrapper/CoreMessageWrapper.java

```diff
@@ -137,247 +175,187 @@ public final String getJMSCorrelationID() throws JMSException {
       message.putBooleanProperty(name, value);
    }
 
-   @Override
-   public final void setByteProperty(String name, byte value) throws JMSException {
+   public final void setByteProperty(String name, byte value)  {
       message.putByteProperty(name, value);
    }
 
-   @Override
-   public final void setShortProperty(String name, short value) throws JMSException {
+   public final void setShortProperty(String name, short value)  {
       message.putShortProperty(name, value);
    }
 
-   @Override
-   public final void setIntProperty(String name, int value) throws JMSException {
+   public final void setIntProperty(String name, int value)  {
       MessageUtil.setIntProperty(message, name, value);
    }
 
-   @Override
-   public final void setLongProperty(String name, long value) throws JMSException {
+   public final void setLongProperty(String name, long value)  {
       MessageUtil.setLongProperty(message, name, value);
    }
 
-   @Override
-   public final void setFloatProperty(String name, float value) throws JMSException {
+   public final void setFloatProperty(String name, float value)  {
       message.putFloatProperty(name, value);
    }
 
-   @Override
-   public final void setDoubleProperty(String name, double value) throws JMSException {
+   public final void setDoubleProperty(String name, double value)  {
       message.putDoubleProperty(name, value);
    }
 
-   @Override
-   public final void setStringProperty(String name, String value) throws JMSException {
+   public final void setStringProperty(String name, String value)  {
       MessageUtil.setStringProperty(message, name, value);
    }
 
-   @Override
-   public final void setObjectProperty(String name, Object value) throws JMSException {
+   public final void setObjectProperty(String name, Object value) {
       MessageUtil.setObjectProperty(message, name, value);
    }
 
-   @Override
-   public final void acknowledge() throws JMSException {
+   public final void acknowledge() {
       // no op
    }
 
-   @Override
-   public void clearBody() throws JMSException {
+   public void clearBody() {
       message.getBodyBuffer().clear();
    }
 
-   @Override
-   public final <T> T getBody(Class<T> c) throws JMSException {
+   public final <T> T getBody(Class<T> c)  {
       // no op.. jms2 not used on the conversion
       return null;
    }
 
    /**
     * Encode the body into the internal message
     */
-   public void encode() throws Exception {
+   public void encode() {
       if (!message.isLargeMessage()) {
          message.getBodyBuffer().resetReaderIndex();
       }
    }
 
-   public void decode() throws Exception {
+   public void decode() {
       if (!message.isLargeMessage()) {
          message.getBodyBuffer().resetReaderIndex();
       }
    }
 
-   @Override
-   public final boolean isBodyAssignableTo(Class c) throws JMSException {
+   public final boolean isBodyAssignableTo(Class c)  {
```

Comment:
If this isnt used and no longer impl interface...would be better to remove. No point having a no op method in a concrete class with no interface contract